ports/windows/variants/standard/mpconfigvariant.props

```diff
@@ -1,3 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="14.0">
+  <PropertyGroup>
+    <FrozenManifest>$(PyVariantDir)manifest.py</FrozenManifest>
```

Comment:
I suggest using `$(PyWinDir)\variants\manifest.py`, then the file `ports/windows/variants/standard/manifest.py` isn't needed.